### PR TITLE
[FrameworkBundle] Name converter of Serializer

### PR DESCRIPTION
**Pull request by dunglas**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes symfony/symfony#14987 |
| Applies to | 2.8 |
| Fixed tickets | n/a |

### CODE REVIEW

File: reference/configuration/framework.rst [hunk 1]
```diff
@@ -1478,7 +1483,45 @@ enabled
...
+
+    For more information, see :ref:`cookbook-serializer-using-serialization-groups-annotations`.
+
+name_converter
+..............
+
+**type**: ``string``
+
+The name converter to use.
+The :class:`Symfony\\Component\\Serializer\\NameConverter\\CamelCaseToSnakeCaseNameConverter` name converter can enabled by using the ``serializer.name_converter.camel_case_to_snake_case``
```
Comment (WouterJ) [inline reference/configuration/framework.rst]: this line has to be broken after the first word that crosses the 72th character

File: reference/configuration/framework.rst [hunk 2]
```diff
@@ -1478,7 +1483,45 @@ enabled
...
+value.
+
+.. seealso::
+
+    For more information, see :ref:`component-serializer-converting-property-names-when-serializing-and-deserializing`.
```
Comment (WouterJ) [inline reference/configuration/framework.rst]: same here (the ref should be placed on its own line). I would also prefer if the ref pointer can be a little bit smaller.

**Comment by xabbuh** [comment]
The code PR has been merged.

**Comment by dunglas** [comment]
Should be ok now.

**Comment by dunglas** [comment]
@xabbuh @WouterJ any update on this one?

**Comment by WouterJ** [comment]
Sorry for the delay @dunglas. Thank you for documenting this new setting. I've merged your PR into the 2.8 version of the docs.

I've also added a versionadded directive, indicating this setting is new in 2.8.

**Pull request closed without merging**